buildSrc/src/main/java/com/uber/okbuck/core/model/jvm/TestOptions.java

```diff
@@ -2,22 +2,23 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.TreeMap;
 
 public final class TestOptions {
 
   private final List<String> jvmArgs;
-  private final Map<String, Object> env;
+  private final TreeMap<String, Object> env;
 
   public TestOptions(List<String> jvmArgs, Map<String, Object> env) {
     this.jvmArgs = jvmArgs;
-    this.env = env;
+    this.env = new TreeMap<>(env);
   }
 
   public List<String> getJvmArgs() {
     return jvmArgs;
   }
 
-  public Map<String, Object> getEnv() {
+  public TreeMap<String, Object> getEnv() {
```

Comment:
Do we need to surface the underlying type on this getter? Looks like it could remain a Map.

Reply:
true, I will change that.